### PR TITLE
Add challenge in confirm

### PR DESCRIPTION
**Pull request by thenav56**
* Fix tabular select input bug. contact @AdityaKhatri for more info.

### CODE REVIEW

File: components/Action/ConfirmOnClick/index.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        static propTypes = propTypes;
+        static defaultProps = defaultProps;
+
+        constructor(props) {
+            super(props);
+
+            this.state = {
+                showConfirm: false,
+                challengeTextInputValue: '',
+                challengeSuccess: false,
```
Comment (tnagorra) [inline components/Action/ConfirmOnClick/index.js]: No need to maintain separate state for challengeSuccess

File: components/Action/ConfirmOnClick/index.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                                        confirmationMessage
+                                        : (
+                                            <p>
+                                                {confirmationMessage}
+                                            </p>
+                                        )
+                                }
+                                { hasChallenge &&
+                                    <div className={styles.challengeForm} >
+                                        <span>{challengeLabel}</span>
```
Comment (tnagorra) [inline components/Action/ConfirmOnClick/index.js:124]: On newline please

File: components/Action/ConfirmOnClick/index.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                        <Confirm
+                            show
+                            disabled={confirmDisabled}
+                            onClose={this.handleModalClose}
+                            title={confirmationTitle}
+                            autoFocus={!hasChallenge}
+                        >
+                            <div className={styles.confirmContent} >
+                                {
+                                    isConfirmationMessageNode ?
```
Comment (tnagorra) [inline components/Action/ConfirmOnClick/index.js]: Better to format it as
```
    isConfirmationMessageNode
        ? (...)
        : (...)
```

File: components/Action/ConfirmOnClick/index.js (hunk 1 above)
Comment (thenav56) [reply components/Action/ConfirmOnClick/index.js]: Fixed!

File: components/Action/ConfirmOnClick/index.js (hunk 2 above)
Comment (thenav56) [reply components/Action/ConfirmOnClick/index.js:124]: Done!

File: components/Action/ConfirmOnClick/index.js (hunk 3 above)
Comment (thenav56) [reply components/Action/ConfirmOnClick/index.js]: Done!

File: components/Action/ConfirmOnClick/index.js (hunk 2 above)
Comment (tnagorra) [reply components/Action/ConfirmOnClick/index.js:124]: Huh? I don't see it in a newline

**Pull request merged**